cartoframes/data/registry/query_dataset.py

```diff
@@ -57,6 +58,21 @@ def compute_geom_type(self):
         """Compute the geometry type from the data"""
         return self._get_geom_type(self._query)
 
+    def get_table_names(self):
+        query = "SELECT CDB_QueryTablesText('{}') as tables".format(self._query)
+        result = self._context.execute_query(query)
+        tables = []
+        if result['total_rows'] > 0:
+            # Dataset_info only works with tables without schema
+            for t in result['rows'][0]['tables']:
```

Comment:
I believe this `for` loop can be replaced by a list comprehension:
```python
tables = [table.split('.')[1] if '.' in table else table
          for table in result['rows'][0]['tables']]
```

Reply:
Woah, nice! I see it more difficult, but I know it is the python way, thank you

Reply:
Yeah, it's not exactly clearer but I think it's more efficient / Pythonic. list.appends tend to be 'slow' too.